engine/termops.ml

```diff
@@ -90,7 +90,7 @@ module Internal = struct
 
   let protect f x =
     try f x
-    with e -> str "EXCEPTION: " ++ str (Printexc.to_string e)
+    with e when CErrors.noncritical e -> str "EXCEPTION: " ++ str (Printexc.to_string e)
```

Comment:
IIRC this is for debug printers which may raise anomalies, not sure guarding is a good idea.